src/components/Send/index.tsx

```diff
@@ -202,7 +202,7 @@ export default function Send({ wallet }: SendProps) {
       })
 
       const loadingWalletInfoAndUtxos = reloadCurrentWalletInfo
-        .reloadUtxos({ signal: abortCtrl.signal })
+        .reloadDisplay({ signal: abortCtrl.signal })
```

Comment:
Reloading only `/display` and not `/utxo` is a problem after sending transactions, as the UI is in an inconsistent state: Still displaying UTXOs that were available before the transaction. It needs to reload `/utxo` or both endpoints.

Reply:
So ig we can use `realoadAll` here?